### PR TITLE
Battery Percentage Indicator 

### LINKED ISSUE #95: Battery Percentage Indicator
This will show the battery percentage of the system.
Can I work on this issue? I'm a participant of LGM Soc21.

### PR DESCRIPTION
**Pull request by Jagannath8**
This is a battery indicator app which is used to know battery percentage of the system.

Fixes #95

### CODE REVIEW

**Review by Ayushparikh-code — APPROVED** [review]

File: Battery Indicator/style.css [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+.toggle::before {
+  content: "\f186";
+  font-family: fontAwesome;
+  color: #fff;
+}
+
+.dark .toggle::before {
+  content: "\f185";
+  color: #161623;
+}
```
Comment (chicken-biryani) [inline Battery Indicator/style.css:187]: Add a new line at the end of file.

File: Battery Indicator/script.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+navigator.getBattery().then(function (battery) {
+  percentage.style.width = battery.level * 100 + "%";
+  percent.innerHTML = Math.floor(battery.level * 100) + "%";
+});
+
+let sec = document.querySelector(".sec");
+let toggle = document.querySelector(".toggle");
+toggle.addEventListener("click", function () {
+  sec.classList.toggle("dark");
+});
```
Comment (chicken-biryani) [inline Battery Indicator/script.js:13]: Add a new line at the end of file.

File: Battery Indicator/index.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+              <div class="percentage"></div>
+            </div>
+          </div>
+          <div class="percent">50%</div>
+        </div>
+      </div>
+    </section>
+    <script src="script.js"></script>
+  </body>
+</html>
```
Comment (chicken-biryani) [inline Battery Indicator/index.html:32]: Add a new line at the end of file.

File: Battery Indicator/README.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Battery Indicator App
+
+This is a battery indicator app which is used to know battery percentage of the system.
+
+
+![battery-indicator](https://github.com/Jagannath8/Web-dev-mini-projects/blob/battery/Battery%20Indicator/light.jpg)
+![battery-indicator](https://github.com/Jagannath8/Web-dev-mini-projects/blob/battery/Battery%20Indicator/dark.jpg)
```
Comment (chicken-biryani) [inline Battery Indicator/README.md:7]: Move images to the end. And add Output // Screenshots as a header.

**Review by chicken-biryani — COMMENTED** [review]
Everything looks pretty good in my opinion. Just add your project name and link in index.md. and do the additions/changes as requested.

**Review by chicken-biryani — COMMENTED** [review]
Also add a steps to set up project. Use PR #159 's README as a reference

**Review by harshita2216 — CHANGES_REQUESTED** [review]
Please add this in index.md, as requested by @chicken-biryani. Then we are good to go! 🚀

**Comment by Jagannath8** [comment]
what link I had to provide? @harshita2216 
image

File: Index.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+| [TO-DO List](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/To-Do%20List)                                   | Simple TO-DO List written in HTML, CSS, and JavaScript   |
+| [Weather-App](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/Weather-App)                                   | Simple Weather app written in HTML, CSS, and JavaScript using the APIs for fetching weather and geolocation information                                                                                                              |
+| [News Webapp](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/flask_news_project)                            | A web app for searching news made using flask                                                                                                                                                                                        |
+| Score Keeper                                                                                                                  |A web Page where user can keep a count of their scores and will tell which player won.|
```
Comment (harshita2216) [inline Index.md:51]: Link of Score Keeper? 
Also, why did you create a new file?

**Comment by Jagannath8** [comment]
Can I make a fresh pr?

**Comment by harshita2216** [comment]
> Can I make a fresh pr?

@Jagannath8, sure you may. Please take your time. You need not hurry.

**Comment by chicken-biryani** [comment]
@Jagannath8 can i close this one?

**Comment by Jagannath8** [comment]
yess @chicken-biryani

**Pull request closed without merging**